archaius2-test/src/test/resources/archaiusRuleTest.properties

```diff
@@ -0,0 +1 @@
+fileProperty=present
```

Comment:
Are you intentionally testing that the last property loads when there is no trailing new line?  If so, I would add a comment explaining that fact, otherwise consider adding a newline to get rid of the angry red icons.